src/Cake.Core/Constants.cs

```diff
@@ -21,5 +21,11 @@ public static class Paths
             public const string Addins = "Paths_Addins";
             public const string Modules = "Paths_Modules";
         }
+
+        public static class Reporting
```

Comment:
I don't think we need constants for these two. Would prefer to hard code them in the `CakeEngine` class.